src/Accounts/Accounts/Accounts.csproj

```diff
@@ -29,6 +29,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="1.1.1" />
```

Comment:
Why add new package reference in Account module?

Reply:
This was automatically added after updating the SecurityCenter nuget

Reply:
Removed

Reply:
This was automatically added after updating the SecurityCenter nuget. Removed.